pkg/ccl/backupccl/backup_job.go

```diff
@@ -226,6 +226,12 @@ func backup(
 					TotalEntryCounts:  backupManifest.EntryCounts,
 					RevisionStartTime: backupManifest.RevisionStartTime,
 				})
+
+				// Always write checkpoints to the progress folder, but only write
```

Comment:
nit: I think this comment can either be deleted or moved into writeBackupManifest now since the logic is better contained.